.gitmodules

```diff
@@ -2,7 +2,6 @@
 	path = react/demos/showcase
 	url = https://github.com/pxblue/react-showcase-demo
 	branch = dev
-[submodule "angular/demos/showcase"]
-	path = angular/demos/showcase
+[submodule "angular/angular-showcase-demo"]
```

Comment:
Is there a reason for changing the demo directory path? I kind of like have the consistent folder structure across all of the packages / frameworks.

Reply:
There's no reason, I just used the repo name as the folder name.  I'll update the folder name to be "showcase" for consistency.